src/test/ui/span/impl-wrong-item-for-trait.stderr

```diff
@@ -0,0 +1,56 @@
+error[E0323]: item `bar` is an associated const, which doesn't match its trait `<FooConstForMethod as Foo>`
```

Comment:
After going through the open tickets, I don't think there's one for this, [so I opened one](https://github.com/rust-lang/rust/issues/37618).